api/source/service/AssetService.js

```diff
@@ -209,7 +209,7 @@ exports.queryAssets = async function (inProjection = [], inPredicates = {}, elev
     for (const pair of inPredicates.metadata) {
       const [key, value] = pair.split(/:(.*)/s)
       predicates.statements.push('JSON_CONTAINS(a.metadata, ?, ?)')
-      predicates.binds.push( `"${value}"`,  `$.${key}`)
+      predicates.binds.push( `${JSON.stringify(value)}`,  `$.${key}`)
```

Comment:
Is there a reason you aren't passing `JSON.stringify(value)` directly as the first argument to `predicates.binds.push()` ? Wrapping it in a template literal wrapper looks superfluous.

Reply:
not a *good* reason!